zsh-system-clipboard.zsh

```diff
@@ -41,7 +41,7 @@ case "$OSTYPE" {
 		fi
 		;;
 	linux*|freebsd*)
-		if _zsh_system_clipboard_command_exists wl-copy && [[ -z "$DISPLAY" ]] && [[ -z "$ZSH_SYSTEM_CLIPBOARD_USE_WL_CLIPBOARD" ]]; then
+		if _zsh_system_clipboard_command_exists wl-copy && [[ -z "$DISPLAY" ]] || [[ "$ZSH_SYSTEM_CLIPBOARD_USE_WL_CLIPBOARD" ]]; then
```

Comment:
What is the difference? Maybe I'm missing some knowledge in ZSH booleans..

Reply:
The -z option checks if the variable is null.
My edit makes sure if the variable is used, wayland is used regardless.

Reply:
Could you explain what do you mean by "wayland is used regardless"? What is your scenario? Do you have `$DISPLAY` set in your environment although you use wayland?

Reply:
I mean if ZSH_SYSTEM_CLIPBOARD_USE_WL_CLIPBOARD is set to anything, then wl-clip is used. At least in my case, Sway sets DISPLAY regardless of me exporting it when launching from a tty, so I edited this to make it work.

Reply:
This is most likely down to xorg-xwayland so if I unset it, then I cannot open any X applications.

Reply:
I also noticed that `ZSH_SYSTEM_CLIPBOARD_USE_WL_CLIPBOARD` is ignored. This PR fixes that for me.

Using the `-v` switch in `[[ ... ]]` makes it consistent with [this description](https://github.com/kutsan/zsh-system-clipboard/blob/f9349b1e51a5e9852ee146256790094b70a18964/zsh-system-clipboard.zsh#L103), i.e. setting the variable to the empty string enables it too.
```suggestion
		if _zsh_system_clipboard_command_exists wl-copy && [[ -z "$DISPLAY" ]] || [[ -v ZSH_SYSTEM_CLIPBOARD_USE_WL_CLIPBOARD ]]; then
```